vsfe/src/main/java/com/example/vsfe/item10_11/FixedMovie.java

```diff
@@ -0,0 +1,62 @@
+package com.example.vsfe.item10_11;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.Arrays;
+
+@Getter
+public class FixedMovie {
+    private final String movieName;
+    private final String director;
+    private final String[] actors;
+
+    @Builder
+    private FixedMovie(String movieName, String director, String ... actors) {
+        this.movieName = movieName;
+        this.director = director;
+        this.actors = new String[actors.length];
+        for(int i = 0; i < actors.length; i++)
+            this.actors[i] = actors[i];
+    }
```

Comment:
I think you don't need to explain details like this if you used @Builder annotation. [Link](https://projectlombok.org/features/Builder)

Reply:
Thx! I overlooked @Builder annotation...